cmd/basecli/commands/cmds.go

```diff
@@ -76,6 +83,16 @@ func doSendTx(cmd *cobra.Command, args []string) error {
 	return txcmd.OutputTx(bres)
 }
 
+// WrapChainTx will wrap the tx with a ChainTx from the standard flags
+func WrapChainTx(tx basecoin.Tx) (res basecoin.Tx, err error) {
+	expires := viper.GetInt64(FlagExpires)
```

Comment:
There is no GetUint() in viper.  But yeah, I set the flag, so it should never be minus.

https://godoc.org/github.com/spf13/viper#Viper.Get